templates/dokumenter.html

```diff
@@ -8,14 +8,20 @@
 {% block content %}
 <h1>{{ section.title }}</h1>
 
-
-{% for x in section.subsections %}
-  {% set y =  x | trim_end_matches(pat="/_index.md") %}
-  {% set z =  y | trim_start_matches(pat="dokumenter/") %}
+<!-- Trims the page link into a single word (name of document). -->
```

Comment:
```suggestion
{# Trims the page link into a single word (name of document). #}
```

Det gjør at denne ikke blir synlig på den ferdig bygget siden, der vil den ikke gi mening 😅

Reply:
Gjør ikke både endringen og den originale koden det helt samme? Er ikke begge kommentarer og derfor usynlig på den ferdig bygget nettsiden?

Reply:
`<!-- kommentar -->` er en HTML-kommentar, disse kommentarene er synlig i den rendrete HTML-koden, men den er ikke synlig på selve siden. `{# kommentar #}` er en Tera-kommentar, disse ignoreres når siden bygges og vil ikke være synlig i HTML-koden som sendes til nettleseren.

Grunnen til at jeg sier at kommentaren burde endres til å bruke Tera sin er fordi det ikke finnes en trim-funksjon i HTML, og for folk som ser på HTML-koden vil det ikke gi mening å se en kommentar for noe som ikke finnes 😅